src/mono/wasm/Wasm.Build.Tests/IcuShardingTests.cs

```diff
@@ -0,0 +1,211 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+using System.Collections.Generic;
+
+#nullable enable
+
+namespace Wasm.Build.Tests;
+
+public class IcuShardingTests : BuildTestBase
+{
+    public IcuShardingTests(ITestOutputHelper output, SharedBuildPerTestClassFixture buildContext)
+        : base(output, buildContext) { }
+
+    private static string customIcuPath = Path.Combine(Directory.GetCurrentDirectory().Split("artifacts")[0], "src", "mono", "wasm", "testassets", "icudt_custom.dat");
```

Comment:
This won't work on helix. Use `Path.Combine(BuildEnvironment.TestAssetsPath, "icudt_custom.dat");`.